cli_tools/common/utils/storage/storage_client.go

```diff
@@ -41,29 +41,25 @@ var (
 // Client implements domain.StorageClientInterface. It implements main Storage functions
 // used by image import features.
 type Client struct {
-	ObjectDeleter domain.StorageObjectDeleterInterface
 	StorageClient *storage.Client
 	Logger        logging.LoggerInterface
 	Ctx           context.Context
 	Oic           domain.ObjectIteratorCreatorInterface
+	Ohc           domain.StorageObjectCreatorInterface
```

Comment:
soc? storageObjectCreator? Longer names are more descriptive IMO, but even the short ones should somehow related to the underlying class, which this one doesn't.

This comment might be redundant depending on whether this interface is needed or not.